src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceChannel.cs

```diff
@@ -556,8 +556,7 @@ private void PrepareCall(ProxyOperationRuntime operation, bool oneway, ref Proxy
                 {
                     if ((context != null) && (!context.IsUserContext) && (context.InternalServiceChannel == this))
                     {
-                        throw ExceptionHelper.PlatformNotSupported();
-                        //throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new InvalidOperationException(SR.Format(SR.SFxCallbackRequestReplyInOrder1, typeof(CallbackBehaviorAttribute).Name)));
+                        throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new InvalidOperationException(SR.Format(SR.SFxCallbackRequestReplyInOrder1, typeof(CallbackBehaviorAttribute).Name)));
```

Comment:
Nit: Not necessary but `nameof(CallbackBehaviorAttribute)` will produce the same value as `typeof(CallbackBehaviorAttribute).Name`